magento/magento-cloud-patches
# MAGECLOUD-4982: De-composed magento-cloud-patches fail to apply locally

### Description

Magento-Cloud-Patches fails to run when missing cloud specific files.  Moved file generation to ece-tools
see PR: https://github.com/magento/ece-tools/pull/663
### Fixed Issues (if relevant)

1. MAGECLOUD-4982

### Manual testing scenarios

1. Deploy to cloud, confirm pub/front-static.php is there and correct
2. Run ece-tools patch locally, confirm works
3. Run ece-patch apply locally, confrim works

### Contribution checklist
